catalog/opensearch/catalog-opensearch-endpoint/pom.xml

```diff
@@ -141,6 +141,7 @@
                         <Embed-Dependency>common-system</Embed-Dependency>
                         <Import-Package>
                             META-INF.cxf,
+                            javax.ws.rs.*,
```

Comment:
I'm not completely sure, to be honest. When I went through the poms I brute-forced the dynamic packages into the import directives one pom after the other. The `*` in this one probably would have picked it up. I can check on it later.